pages/desktop/consumer_pages/base.py

```diff
@@ -67,10 +68,12 @@ def set_region(self, region):
 
         self.wait_notification_box_not_visible()
 
-    def login(self, user=None):
-        from pages.fxa import FirefoxAccounts
-        fxa = FirefoxAccounts(self.testsetup)
-        fxa.login_user(user)
+    def login(self, user):
+        credentials = isinstance(user, MockUser) and user
```

Comment:
This does not really make sense. If `isinstance(user, MockUser)` returns `False`, then `credentials` will also be `False`, which will cause an exception at line 75 below. In fact, I am seeing exactly that in 3 of the tests when I run them locally. See below for one example:

```
________________________________________________ TestConsumerPage.test_footer_section_links _________________________________________________

self = <tests.desktop.consumer_pages.test_home_page.TestConsumerPage instance at 0x110718a28>
mozwebqa = <pytest_mozwebqa.pytest_mozwebqa.TestSetup instance at 0x110718638>

    @pytest.mark.sanity
    @pytest.mark.nondestructive
    def test_footer_section_links(self, mozwebqa):
        home_page = Home(mozwebqa)
        home_page.go_to_homepage()

        home_page.header.click_sign_in()
>       home_page.login(user="default")

tests/desktop/consumer_pages/test_home_page.py:142:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pages.desktop.consumer_pages.home.Home object at 0x110711350>, user = 'default'

    def login(self, user):
        credentials = isinstance(user, MockUser) and user
        from fxapom.pages.desktop.sign_in import SignIn
        fxa_login = SignIn(self.testsetup)
>       fxa_login.sign_in(credentials['email'], credentials['password'])
E       TypeError: 'bool' object has no attribute '__getitem__'

pages/desktop/consumer_pages/base.py:78: TypeError
```